CMakeLists.txt

```diff
@@ -482,27 +482,6 @@ else()
 endif()
 set(OSCAP_TEMP_DIR "/tmp" CACHE STRING "use different temporary directory to execute sce scripts (default=/tmp)")
 
-# ---------- RPATHS for linking
-
-# see https://cmake.org/Wiki/CMake_RPATH_handling
-
-# use, i.e. don't skip the full RPATH for the build tree
-set(CMAKE_SKIP_BUILD_RPATH FALSE)
```

Comment:
When reading https://cmake.org/cmake/help/latest/variable/CMAKE_SKIP_BUILD_RPATH.html I think that we should rather set CMAKE_SKIP_BUILD_RPATH to TRUE than removing it completely because the variable doesn't have a default. What do you think?

Reply:
I agree.

Reply:
I have removed the removal and replaced it by replacement of the value.

Reply:
But it has broken all the tests.

Reply:
Hmm, it looks like it breaks the `add_oscap_test_executable` from `tests/CMakeLists.txt`.

Reply:
it worked locally because unfortunately it linked to the system installed openscap :scream: